awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -269,9 +282,18 @@ class JobOutput extends Component {
     this.scrollToRow(0);
   }
 
-  handleScrollLast() {
-    const { remoteRowCount } = this.state;
-    this.scrollToRow(remoteRowCount - 1);
+  async handleScrollLast() {
+    const { job, type } = this.props;
+    const {
+      data: {
+        results: [{ counter }],
+      },
+    } = await JobsAPI.readEvents(job.id, type, {
+      page_size: 1,
+      order_by: '-counter',
+    });
+    this.setState({ remoteRowCount: counter + 1 });
+    this.scrollToRow(counter);
   }
```

Comment:
@ryanpetrello @trahman73 @wenottingham 

One path forward I can see for making this work without a total count is to _turn off scrolling_ in the job output viewer whenever a search filter is applied. 

Applying a search filter to the results would put the output ui into a new mode. In this mode, only the `up`, `down`, `first`, and `last` navigation buttons would be available. There would be no infinite scrolling of filtered job events.

Reply:
**unfiltered job results and running jobs:** Have a virtualized / "infinite" scroll with a range that is adjusted by the total event count (just as it is now in ui_next). We can use the highest event counter as the "total count" in this scenario.

**jobs with search filter applied:** Scrolling works more like it does in current awx. The scrollable range will need to grow as you scroll up or down into new regions, and it'll completely reset if you do a `first` or `last` navigation.